UI/window-basic-main.hpp

```diff
@@ -869,7 +867,10 @@ private slots:
 		return os_atomic_load_bool(&previewProgramMode);
 	}
 
-	inline bool VCamEnabled() const { return vcamEnabled; }
+	inline bool VCamEnabled() const
+	{
+		return vcamEnabled;
+	}
```

Comment:
Please run your code through `clang-format` to make sure that these lines don't get reformatted

Reply:
I did, weirdly it got formatted differently on my machine.

Reply:
This could depend on the version of clang-format you're running. We're using 13, which iirc should be in the current VS Code.
Otherwise, in this case it would probably work to manually undo the changes.

Reply:
Alternatively you could just cherry pick the lines/hunks you want to have changed for the commit instead of the entire file if clang-format still formats parts that you actually don't intend to change.